File: content/en/04-llms/llms/_solutions/02_conversation/02_conversation.R
library(ellmer)

chat <- chat_posit(
  system_prompt = "Answer in as few words as possible."
)

chat$chat("What creates an Anthropic chat in ellmer and chatlas?")
chat$chat("What about OpenAI?")

# Compare with a fresh chat, no system prompt:
chat2 <- chat_posit()
chat2$chat("What about OpenAI?")

File: content/en/04-llms/llms/_solutions/02_conversation/02_conversation.py
import chatlas

chat = chatlas.ChatPosit(
    system_prompt="Answer in as few words as possible."
)
chat.chat("What creates an Anthropic chat in ellmer and chatlas?")
chat.chat("What about OpenAI?")

# Compare with a fresh chat, no system prompt:
chat2 = chatlas.ChatPosit()
chat2.chat("What about OpenAI?")
